README.md

```diff
@@ -1 +1,25 @@
-# hello-microverse
\ No newline at end of file
+# hello-microverse
+# Project Name
+
+> Hello World Microverse Project.
+
+
+## Built With
+
+- HTML/CSS
+- linters
+
+
```

Comment:
- [ ] Kindly add the getting started section, you can use [this template](https://github.com/microverseinc/readme-template) for guide you in implementing it.

Reply:
I have fixed the README section according to the template